gammapy/datasets/tests/test_map.py

```diff
@@ -1052,15 +1052,15 @@ def test_stack_dataset_dataset_on_off():
     dataset_on_off.counts_off += 1
     dataset.stack(dataset_on_off)
 
-    assert_allclose(dataset.background_model.map.data, 0.2)
+    assert_allclose(dataset.background_model.map.data, 0.166667, rtol=1e-3)
```

Comment:
I think this should not have changed (there is no model defined on the `MapDatasetOnOff`). I suppose the issue is connected with the issue in `stack`.

Reply:
This I was a bit confused about - Does it make sense to stack a `MapDatasetOnOff` on a `MapDataset`?